app/controllers/bestuurseenheid/reglementen/index.js

```diff
@@ -0,0 +1,48 @@
+import Controller from '@ember/controller';
+import { tracked } from '@glimmer/tracking';
+import { service } from '@ember/service';
+import { restartableTask } from 'ember-concurrency';
+
+const PAGE_SIZE = 10;
+
+export default class ReglementenIndexController extends Controller {
+  @service store;
+  @tracked from;
+  @tracked to;
+  @tracked page = 0;
+  @tracked bestuurseenheid;
+  @tracked model;
```

Comment:
the model property on a controller is automatically set, and is always tracked. It gets the resolved return value of the model hook in the route class

Reply:
the same also here, I tried to stay consistent with other controllers: https://github.com/lblod/frontend-gelinkt-notuleren-publicatie/blob/master/app/controllers/bestuurseenheid/index.js#L71 . I will have a look now